jwst/ami/ami_analyze_step.py

```diff
@@ -180,11 +184,19 @@ def process(self, input):
             # If there's a user-defined bandpass or affine, handle it
             if bandpass is not None:
                 bandpass = self.override_bandpass()
-
             if affine2d is not None:
-                # if it is None, handled in apply_LG_plus
-                affine2d = self.override_affine2d()
-
+                if affine2d == 'commissioning':
+                    affine2d = utils.Affine2d(mx=9.92820e-01,
+                        my=9.98540e-01,
+                        sx=6.18605e-03,
+                        sy=-7.27008e-03,
+                        xo=0,
+                        yo=0,
+                        name='commissioning')
```

Comment:
Suggest moving these constants to the top of the file and giving them capitalized variable names to conform more closely with PEP8: https://peps.python.org/pep-0008/#constants